src/test/java/seedu/address/logic/commands/customer/CustomerAddCommandTest.java

```diff
@@ -31,57 +31,58 @@
 import seedu.address.testutil.PersonBuilder;
 import seedu.address.ui.ListItem;
 
-public class AddCommandTest {
+public class CustomerAddCommandTest {
```

Comment:
Is it possible to add a test for wrong phone number length? Both in `customer add` and `customer edit` so as to prevent regression in the future.

Reply:
As the checking of phone number field is handled by `CustomerAddCommandParser` and `Phone`, I was unable to create a `customer` with invalid phone number to test in `customer add`. I have added more invalid phone length test cases in `CustomerAddCommandParserTest` instead.